### ISSUE TITLE
only 200 of 263 google group members are loaded 

### ISSUE THREAD

**Issue by dalepaul-RL** [slashdevops/idp-scim-sync#87]
**Describe the bug**
Only 200 users of 263 members of a google group are sync'd 
All appropriate users are added to Identity Center user base, but adding the members to groups seems to be where the problem lies.

**To Reproduce**
Sync a group containing more than 200 members. You will see two "batches" of 100 users each are retrieved. Syncing stops after the second batch with no error. If you add a single member to the large group, the behavior is to add the new user if it falls under the 200 limit, and a single user is then removed from the group if they fall over the 200 limit.

**Expected behavior**
all users in a group are added

**Additional context**
I have not found any quota limits in either the AWS SCIM, AWS or Google that would prevent this. I also do not know why the "3rd" page of users is not coming in, i.e. nextpage token not present, or hard carded value in the source). Execution traces did show that group members were done in batch size of 100 per request.

Thanks in advance

**Comment by dalepaul-RL** [comment]
I tracked this down to a bug in the Go Admin SDK. https://issuetracker.google.com/247106193
The issue is when adding fields to the URL for a shortened response results in the nextPageToken not being sent.
Since this is not a bug in this code base, and I am not familiar with the configuration parameters to know if this code base can no use the shortened response format, you may close this issue.
Thanks!

**Comment by dalepaul-RL** [comment]
FIX:
please add "nextPageToken" to the fields list in google.go lines 17-19
```
	groupsRequiredFields    googleapi.Field = "nextPageToken, groups(id,name,email,etag)"
	membersRequiredFields   googleapi.Field = "nextPageToken, members(id,email,status,type,etag)"
	listUsersRequiredFields googleapi.Field = "nextPageToken, users(id,name,primaryEmail,suspended,etag,emails)"
```

This will resolve the issue of your program not paginating results from the Google API

**Comment by christiangda** [comment]
Hi @dalepaul-RL sorry for being late here and thank you very much for taking care of this.

I will add the field required soon and I will release a new version.

**Comment by dalepaul-RL** [comment]
Glad to help, and thank you for getting this corrected in a new version. I look forward to using your app to sync out Identity Center in production soon. Best wishes.

**Comment by christiangda** [comment]
fixed in PR #90
